contributed_definitions/nyaml/NXapm_compositionspace_results.yaml

```diff
@@ -1,19 +1,21 @@
 category: application
 doc: |
-  Results of a run with Alaukik Saxena's composition space tool.
+  Application definition for results of the CompositionSpace tool used in atom probe.
   
-  This is an initial draft application definition for the common NFDI-MatWerk,
-  FAIRmat infrastructure use case IUC09 how to improve the organization and
-  results storage of the composition space tool and make these data at the
-  same time directly understandable for research data management systems
-  like NOMAD Oasis.
+  * `A. Saxena et al. <https://www.github.com/eisenforschung/CompositionSpacetool.git>`_
+  
+  This is a draft application definition for the common NFDI-MatWerk/FAIRmat infrastructure
+  use case IUC09 that explores how to improve the organization and results storage of the
+  CompositionSpace tool.
 symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
   d: |
```

Comment:
I would suggest to use better symbols (like `grid_dim`, `n_voxels`).

Reply:
Accepted and implemented